static/js/recent_topics.js

```diff
@@ -600,6 +600,9 @@ export function change_focused_element(e, input_key) {
                 current_focus_elem = $("#recent_topics_search");
                 return true;
             case "escape":
+                if (current_focus_elem === "table") {
+                    return false;
+                }
```

Comment:
Can you expand the commit message to explain more clearly what this is for.  I'm guessing it's to allow Esc to take you to another view if "Recent topics" is not the default view?

Reply:
Yes.